internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -311,24 +320,10 @@ func (r *BMACReconciler) Reconcile(origCtx context.Context, req ctrl.Request) (c
 		return res.Result()
 	}
 
-	if !r.ConvergedFlowEnabled {
-		// After the agent has started installation, Ironic should not manage the host.
-		// Adding the detached annotation to the BMH stops Ironic from managing it.
-		result = r.addBMHDetachedAnnotationIfHostIsRebooting(log, bmh, agent)
-		if res := r.handleReconcileResult(ctx, log, result, bmh); res != nil {
-			return res.Result()
-		}
-	}
-
-	result = r.reconcileSpokeBMH(ctx, log, bmh, agent)
-	if result.Dirty() {
-		err := r.Client.Update(ctx, bmh)
-		if err != nil {
-			log.WithError(err).Errorf("Error adding BMH detached annotation after creating spoke BMH")
-			return reconcileError{err: err}.Result()
-		}
+	result = r.reconcileDay2SpokeBMH(ctx, log, bmh, agent)
```

Comment:
Why day2?

Reply:
Maybe misleading but the comment in the function mentions it's only for day 2 hosts. Maybe that should mean for installed hosts? https://github.com/openshift/assisted-service/pull/7749/files#diff-86415614c6ed40c56c5ca2b5f751a65746ef5c83b61e6f021a4f5d817e4425baR1041-R1044

```
	if !installed {
		log.Infof("Skipping spoke BareMetalHost reconcile for agent %s/%s since it's not day2.", agent.Name, agent.Namespace)
		return reconcileComplete{}
	}
```